### PR TITLE
Allow payment nonce

### PR DESCRIPTION
**Pull request by captainbeardo**
## Ticket

https://jira.bqsoft.com/browse/ATOM-6477

## What this does

Allows passing in a nonce to create a payment instead of a token.

## Why we did this

Client side credit card management

## Overall, I feel this way about this code

Neat

### CODE REVIEW

File: lib/conduit/braintree/actions/authorize_transaction.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 module Conduit::Driver::Braintree
   class AuthorizeTransaction < Base
-    required_attributes :amount, :token
+    required_attributes :amount, [:token, :payment_method_nonce]
```
Comment (pedromartinez) [inline lib/conduit/braintree/actions/authorize_transaction.rb]: Isn't a nonce used to store the card?

**Comment by coreypurcell** [comment]
Why do we need to do this?

File: lib/conduit/braintree/actions/authorize_transaction.rb (hunk 1 above)
Comment (captainbeardo) [inline lib/conduit/braintree/actions/authorize_transaction.rb]: Are we making one-time payments through magnetic? Or just managing credit cards?

Comment (pedromartinez) [inline lib/conduit/braintree/actions/authorize_transaction.rb]: Just managing credit cards.

**Pull request closed without merging**